mmv1/products/firebasedatabase/terraform.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Instance: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ['projects/{{project}}/locations/{{region}}/instances/{{instance_id}}', '{{region}}/{{instance_id}}']
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "firebase_database_instance"
+        min_version: "beta"
+        primary_resource_id: "basic"
+        vars:
+          instance_id: "active-db"
```

Comment:
Oops, one last minute comment: could you please prefix these `instance_id`s and the `project_id` further down with `tf-test`? That makes it easier for us to clean them up from the test environment.

Reply:
https://github.com/modular-magician/terraform-provider-google-beta/blob/7dbbbb260f810317751d3f7e7355a5b2594d3d60/google-beta/resource_firebase_database_instance_generated_test.go

I think the code generation already adds those prefix for us.

Reply:
Oh wow, I wasn't aware of that, thanks!